.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: CI
+on:
+  push:
+  pull_request:
+
+jobs:
+  deno-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+      - name: Check types
+        run: shopt -s globstar; deno check denops/**/*.ts
```

Running `deno check` with specific directory is supported in latest Deno.
```suggestion
        run: deno check denops
```

---

Thank you.